src/runtime_src/core/tools/xbmgmt2/flash/flasher.cpp

```diff
@@ -59,6 +59,9 @@ Flasher::E_FlasherType Flasher::typeStr_to_E_FlasherType(const std::string& type
     else if (typeStr.compare("qspi_versal") == 0) {
         type = E_FlasherType::QSPIVERSAL;
     }
+    else if (typeStr.compare("ospi_xgq") == 0) {
+        type = E_FlasherType::OSPI_XGQ;
```

Comment:
In general our convention is no curly brace for single statement conditional, but I see this is just following what was already done in parts of this messy file.

Reply:
There are many codes like this in this file. Should I follow the existing pattern or make it special case for not having {} for this change?

Reply:
No, just leave as is.